README.md

```diff
@@ -46,32 +46,6 @@ Our main website is https://polykey.io
 npm install --save polykey
 ```
 
-### Nix/NixOS
-
-Building the releases:
-
-```sh
-nix-build ./release.nix --attr application
-nix-build ./release.nix --attr docker
-nix-build ./release.nix --attr package.linux.x64.elf
-nix-build ./release.nix --attr package.windows.x64.exe
-nix-build ./release.nix --attr package.macos.x64.macho
-```
-
-Install into Nix user profile:
-
-```sh
-nix-env -f ./release.nix --install --attr application
-```
-
-### Docker
-
-Install into Docker:
-
-```sh
-docker load --input "$(nix-build ./release.nix --attr docker)"
-```
-
```

Comment:
Can you make sure that these README notes and related information goes into `Polykey-CLI`.

Reply:
This has been moved over.